src/main/java/life/genny/utils/BaseEntityUtils.java

```diff
@@ -2039,54 +2000,32 @@ public void setServiceToken(GennyToken serviceToken) {
 		}
 		hql += " where ";
 
-		if (attributeFilterCode1 != null) {
-			hql += " ea.baseEntityCode=eb.baseEntityCode ";
-			if (beFilter1 != null) {
-				hql += " and (ea.baseEntityCode like '" + beFilter1 + "'  ";
-			}
-		} else {
-			if (beFilter1 != null) {
-				hql += " (ea.baseEntityCode like '" + beFilter1 + "'  ";
+		if (beFilters.size() > 0) {
+			hql += " ( ";
+			for (int i = 0; i < beFilters.size(); i++) {
+				if (i > 0) {
```

Comment:
StringBuild could be considered to add the String 

BTW, looks Like it’s a big change, does any test cases to guarantee it’s all good?

Roger Xia